.npmrc

```diff
@@ -0,0 +1,12 @@
+# Hoist these dependencies for workflow tools (linter, typechecker, etc.)
+public-hoist-pattern[]=*prettier*
+public-hoist-pattern[]=@docusaurus/types
+public-hoist-pattern[]=@docusaurus/theme-common
+public-hoist-pattern[]=@types/node
+
+# Hoist these dependencies because development builds in .docusaurus/
+# assume they will be top-level for hot-reloading
+# TODO: re-evaluate whether some of these can become genuine top-level
+# dependencies if we end up making custom React components
+public-hoist-pattern[]=react
+public-hoist-pattern[]=@mdx-js/react
```

Comment:
add newline to eof

Reply:
No longer relevant, file removed in https://github.com/comcode-org/hackmud_wiki/pull/2/commits/41a76ef480e7c28908dd5e6d9c3bfdca5c107b3d.